res/values/arrays.xml

```diff
@@ -27,16 +28,19 @@
     <item>QWERTZ</item>
     <item>Bulgarian (Phonetic Traditional)</item>
     <item>Dvorak</item>
+    <item>Neo 2</item>
   </string-array>
   <string-array name="pref_programming_layout_values">
     <item>none</item>
     <item>qwerty</item>
     <item>dvorak</item>
+    <item>neo2</item>
```

Comment:
Is it actually a programming layout ?

Programming layouts are made for people that need a different layout for programming and for typing in their usual language. (makes most sense for non-Latin languages users) Is it plausible that someone would want to use `neo2` when programming but an other layout the rest of the time ?

Reply:
Hm, not really. I don't think non German people will benefit from it. Should I remove it from the list then?

Reply:
Yes, let's remove it from that list.

Reply:
Done. I also added the new accents (accent_ogonek definitely is not present on the keyboard)